WordPress/Classes/ViewRelated/Media/MediaItemViewController.swift

```diff
@@ -75,39 +71,23 @@ class MediaItemViewController: UITableViewController {
         }
 
         viewModel = ImmuTable(sections: [
-            ImmuTableSection(rows: [ headerRow ]),
             ImmuTableSection(headerText: nil, rows: mediaInfoRows, footerText: nil),
             ImmuTableSection(headerText: nil, rows: metadataRows, footerText: nil)
         ])
+
+        headerView.heightAnchor.constraint(greaterThanOrEqualToConstant: 280).isActive = true
+        headerMaxHeightConstraint.isActive = true
+        headerView.configure(with: media)
+        tableView.tableHeaderView = headerView
+
+        headerView.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(didTapHeaderView)))
     }
 
-    private var headerRow: ImmuTableRow {
-        switch media.mediaType {
-        case .image, .video:
-            return MediaImageRow(media: media, action: { [weak self] row in
-                guard let media = self?.media else { return }
-
-                switch media.mediaType {
-                case .image:
-                    if self?.isMediaLoaded() == true {
-                        self?.presentImageViewControllerForMedia()
-                    }
-                case .video:
-                    self?.presentVideoViewControllerForMedia()
-                default: break
-                }
-            })
-        default:
-            return MediaDocumentRow(media: media, action: { [weak self] _ in
-                guard let media = self?.media else { return }
-
-                // We're currently not presenting previews for audio until
-                // we can resolve an auth issue. @frosty 2017-05-02
-                if media.mediaType != .audio {
-                    self?.presentDocumentViewControllerForMedia()
-                }
-            })
-        }
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        headerMaxHeightConstraint.constant = view.bounds.height * 0.75
```

Comment:
I forgot `multiplier` was a thing. Unfortunately, it doesn't seem to work in this case maybe because the constraint system doesn't receive an updated view.height value until after the `viewDidLayoutSubviews` is completed, which is too late. You can reproduce it by running it on iPad and rotating the device from portrait to landscape.